setup.py

```diff
@@ -41,6 +41,7 @@ def run(self):
     url='https://github.com/pytest-dev/pytest-localserver',
 
     packages=['pytest_localserver'],
+    python_requires='>=3.4',
```

Comment:
Seems like this should be:
```suggestion
    python_requires='>3.4',
```

Reply:
In the current iteration of the PR, it was intentional to include Python 3.4 in the set of supported versions. My reasoning was that, since #21 uses the `enum` module, which is only included in Python 3.4 and up, we _know_ that the package won't work on 3.3 after that gets merged, whereas we don't have any similar reason to expect that it wouldn't work on 3.4. So I was thinking I'd just remove support for 3.3 in this PR, in anticipation of #21 getting merged later. That way, if anyone is still stuck using Python 3.4 and wants to install this package, they can still try their luck with it, whereas I think changing `python_requires` to exclude 3.4 would actually prevent it from being installed on that version.

Anyway, in retrospect, it probably does make more sense make it so that only 3.5+ is officially supported because those are the only versions we're testing, so I'm on board with that.

BTW maybe this is just me, but I'd prefer to express it as

```suggestion
    python_requires='>=3.5',
```

because if someone briefly glances at the entry and sees the version number but misses or misreads the comparison operator, then they'll come away with a version that the package actually does work on (3.5). Also, I'm not totally sure offhand if `>3.4` would allow 3.4.1 and so on. What do you think?

Reply:
I pushed fixup commits to implement dropping support for 3.4 as well.